examples/Exploring-different-models.ipynb

```diff
@@ -0,0 +1,802 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #19.    from merlin.schema.io.tensorflow_metadata import TensorflowMetadata
</pre><p>Nit that's not about the notebook, but the API.  If we have <code>merlin.models.tf</code>, then maybe we should have <code>merlin.schema.io.tf_metadata</code>.  </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/NVIDIA-Merlin/models/pull/256/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='4'/>

Reply:
@mikemckiernan thanks. You can mention that in our models slack channel. But this will affect many code :)